src/java.base/share/classes/java/io/ObjectInputStream.java

```diff
@@ -371,13 +371,13 @@ private static class Logging {
      * read* requests
      */
 
-    /* ClassCache Entry for caching class.forName results upon enableClassCaching */
-    private static final ClassCache classCache;
+    /* ClassNameCache Entry for caching class.forName results upon enableClassCaching */
+    private static final ClassNameCache classNameCache;
```

Comment:
This field should probably also be renamed in jdknext.

Reply:
Actually, better names might be `ClassByNameCache` and `classByNameCache`.

Reply:
Whatever we decide should be propagated to newer versions so the differences are minimal.